src/components/vr-stereo/vr-stereo.tsx

```diff
@@ -38,7 +40,40 @@ const COMPONENT_NAME = 'VrStereo';
 @withText({
   vrStereoText: 'controls.vrStereo'
 })
-class VrStereo extends Component<any, any> {
+class VrStereo extends Component<any, any> implements IconComponent {
+  /**
+   * Creates an instance of PictureInPicture.
+   * @memberof VrStereo
+   */
+  constructor(props: any) {
+    super(props);
+    registerToBottomBar(COMPONENT_NAME, props.player, () => this.registerComponent());
+  }
+
+  registerComponent(): any {
+    return {
+      ariaLabel: () => this.getComponentText(),
+      displayName: COMPONENT_NAME,
+      order: 5,
+      svgIcon: () => this.getSvgIcon(),
+      onClick: () => this.onClick(),
+      component: () => {
+        return getComponent({...this.props, classNames: [style.upperBarIcon]});
+      },
+      shouldHandleOnClick: false
```

Comment:
why the onClick is not sufficient to deduce is it should be handheld

Reply:
as explained here- https://github.com/kaltura/playkit-js-ui-managers/pull/59#discussion_r1673482830